### ISSUE TITLE
Internet Explorer 11: PrimaryButton text is not visible in High Contrast mode

### ISSUE THREAD

**Issue by prashkan90** [OfficeDev/office-ui-fabric-react#9345]
## Actual behavior (IE 11):
![image](https://user-images.githubusercontent.com/10136561/58925525-b353b980-86fc-11e9-90de-5aefed49f50c.png)

## Expected behavior (Edge):
![image](https://user-images.githubusercontent.com/10136561/58925748-7f2cc880-86fd-11e9-9950-c5259227613b.png)

Relevant CSS property:
![image](https://user-images.githubusercontent.com/10136561/58925705-5573a180-86fd-11e9-85ae-971356799c3b.png)

**Comment by aneeshack4** [comment]
@prashkan90 This doesn't repro on IE11 on the Primary Button on the Fabric docs site: You can see the version of IE 11 in this screenshot. Is your version different? Or maybe you are adding custom styling that's causing this?
![image](https://user-images.githubusercontent.com/4161791/58995800-8067ff00-87aa-11e9-9bdd-8ce459e8f8ff.png)

**Comment by prashkan90** [comment]
@aneeshack4 Looks like I was using an older version of IE. Thanks for checking. I'll close the issue.
